libc-bottom-half/crt/crt1.c

```diff
@@ -2,23 +2,17 @@
 extern void __wasm_call_ctors(void);
 extern int __original_main(void);
 extern void __wasm_call_dtors(void);
+_Noreturn void exit(int);
```

Comment:
Since this is public API can you just include unistd instead?

Reply:
Done.